java/printing/resources/geoserver/print/config.yaml

```diff
@@ -155,6 +155,110 @@ layouts:
               maxSize: 200
               type: 'bar sub'
               intervals: 5
+  A4_long_profile :
```

Comment:
@MV88 why this config.yaml is needed? Is this a typo? I know we manage the printing completely client side therefore it should not be necessary.

Reply:
it's a refuse. not needed going to remove